core/shared/src/main/scala/chartreuse/component/AnnotationPosition.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023 Creative Scala
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package chartreuse.component
+
+import doodle.core.*
+
+final case class AnnotationPosition(
+    landmark: Landmark,
+    arrowAngle: Angle,
+    toPoint: (Point, Double) => Point
```

Comment:
It's not obvious what this function is doing. I think it could use a bit of documentation.